src/controllers/teams.controller/deleteTeamMembers/index.js

```diff
@@ -15,54 +29,33 @@ const deleteTeamMembers = catchAsync(async (req, res) => {
 
   const { user_id, cognito_sub, team_id } = req.body;
 
-  const variables = {
-    team_id,
-    user_id,
-    cognito_sub,
-  };
+  const variables = { team_id, user_id, cognito_sub };
 
   const response1 = await Hasura(checkIfCanDelete, variables);
+  const creatorId = response1.result.data.members[0].team.creator_id;
+  const currentUserId = response1.result.data.members[0].team.user.id;
 
-  if (response1.result.data.admins.length == 0) {
-    return res.status(401).json({
-      success: false,
-      errorCode: 'Forbidden',
-      errorMessage: 'You are not an admin of this team.',
-      data: null,
-    });
+  const variables2 = { user_id, team_id };
+
+  if (response1.result.data.admins.length === 0) {
+    return res.status(401).json(errorResponse('Forbidden', 'You are not an admin of this team.'));
   }
 
-  if (response1.result.data.members.length == 0) {
-    return res.status(400).json({
-      success: false,
-      errorCode: 'Forbidden',
-      errorMessage: 'Given user is not a member of this team.',
-      data: null,
-    });
+  if (response1.result.data.members.length === 0) {
+    return res.status(400).json(errorResponse('Forbidden', 'Given user is not a member of this team.'));
   }
 
   if (response1.result.data.members[0].admin) {
```

Comment:
Please simplify this if block further using boolean operators. See if you can avoid having deleteTeamMember queries at multiple places. Only one, on line 57 should be enough, that can be reached if all checks are passed.